### ISSUE TITLE
Rating star: NVDA + Chrome not announcing the readonly rating 

### ISSUE THREAD

**Issue by Hardhik** [OfficeDev/office-ui-fabric-react#9787]
# Actual behavior: 

For NVDA + Chrome Testing:
When we bring focus to the Rating control in readonly mode, the NVDA reads out control as "status" (the focused element here is the div created from FocusZone which doesn't have the aria-label)

The div with class "ms-Rating-star" has the aria-label but the FocusZone has the tabindex=0
This causes the NVDA to read it as "blank" instead of something like "Rating is 4 of 5."

## Expected behavior:
NVDA should read out the rating of the star.

Code where this issue can be fixed:

https://github.com/OfficeDev/office-ui-fabric-react/blob/6ff7873e60436513ce7c3e4f39982a461fa5c86f/packages/office-ui-fabric-react/src/components/Rating/Rating.base.tsx#L127

Two possible solutions:
1) The aria-label here needs to be moved to the FocusZone
2) The tabindex on FocusZone to be moved to it's parent div.

PS: Tried this locally by inspecting element and it works as expected by changing it.
